### ISSUE TITLE
Dask release 2024.12.0 changed `dsk` keyword in Blockwise class this breaks AwkwardBlockwiseLayer `__init__`

### ISSUE THREAD

**Issue by jpwgnr** [dask-contrib/dask-awkward#557]
Hi,
the change [here](https://github.com/dask/dask/commit/652829d464aa1cb1dfe5134c8504473e9ce12aba#diff-b2b064ba4d14c2c4d3c14bb57d0cda3849f084625015bb61f2ec1f7ffe8195ccL375) in dask version 2024.12.0 breaks AwkwardBlockWiseLayer `super().__init__` function:

![image](https://github.com/user-attachments/assets/e820de62-8466-4bcd-8c96-869131ee48b4)

I am no expert whatsoever just tried to run most recent version and saw the break in the inheritance of the class.

**Comment by martindurant** [comment]
@lgray , will #556 fix this? We have no way to add a dask version constraint to released versions...

**Comment by lgray** [comment]
Yeah once I get the details worked out #556 will take care of this. For now, @jpwgnr please use an older version of dask.

**Comment by jpwgnr** [comment]
sure all good, just wanted to let you know. :)

**Comment by lgray** [comment]
Thanks! The nightly tests in coffea also picked it up, but good to have other confirmation!

**Comment by Saransh-cpp** [comment]
+1 from vector

**Comment by jpwgnr** [comment]
Hi,
I think this is fixed so I close the issue. 
At least with version `2025.2.0` of `dask` and `dask-awkward` everything looks smooth now.
Thanks!
